.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
```

Comment:
Is this right?

Reply:
Yea, it’s so that prs against branches other than main will trigger CI (for example, against a 0.15 branch). We’ll want to update CI elsewhere too eventually, but it’s not a big deal.